tests/stats/test_stats.py

```diff
@@ -343,11 +344,13 @@ def setup_client_side_stats(self):
             weblog.get("/")
 
     @missing_feature(
-        context.library in ("cpp", "cpp_httpd", "cpp_nginx", "dotnet", "java", "nodejs", "php", "python", "ruby"),
+        context.library in ("cpp", "cpp_httpd", "cpp_nginx", "dotnet", "nodejs", "php", "python", "ruby"),
         reason="Tracers have not implemented this feature yet.",
     )
     def test_client_side_stats(self):
         """Test that client-side stats are properly bucketed in 10-second intervals"""
+        # wait for 10 seconds to be sure all the buckets are flushed (better than be flaky)
+        time.sleep(10)
```

Comment:
At this point, the infra is down, so it's pointless to wait.

you need to increase `library_interface_timeout` or `agent_interface_timeout` to achieve this. Feel free to ping me if you need more details